tests/objects/test_glyph.py

```diff
@@ -0,0 +1,75 @@
+from ufoLib2.objects import Anchor, Component, Glyph, Guideline, Image
+
+
+def test_copyDataFromGlyph(ufo_UbuTestData):
```

Comment:
Is it worth adding additional `assert`s to check that the data is independent and writing to one lib will not affect the other?

Reply:
I asserted that they are different objects using `is not` operator